test/Basics/Labels.js

```diff
@@ -338,6 +338,21 @@ testNoSyntaxErrorWithDanglingStaticAfterLabel=
 testRuntimeErrorWithDanglingStaticAfterLabel=
     `L: static
     x = 0`
+
+testSyntaxErrorWithContinueAfterLabel=
+    `function fn() {
+        L:
+            continue L;
+    }
+    fn1()`
+
+testReferenceErrorWithBreakAfterLabel=

```

Comment:
Does it make sense to make this into a test case which doesn't throw? Use `assert.doesNotThrow` below? The reference error we're testing for here has nothing to do with the label binding.